stac_fastapi/api/stac_fastapi/api/openapi.py

```diff
@@ -1,68 +1,36 @@
 """openapi."""
 from fastapi import FastAPI
-from fastapi.openapi.utils import get_openapi
 from starlette.requests import Request
-from starlette.responses import JSONResponse
-
-from stac_fastapi.api.config import ApiExtensions
-from stac_fastapi.types.config import ApiSettings
-
-
-class VndOaiResponse(JSONResponse):
```

Comment:
Should be deprecated instead of removed, so we can include this fix in a MINOR or PATCH release.